docs/troubleshooting/supported-browsers.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Supported Browsers List
+image: "/docs/images/docs-meta-cards/troubleshoot-card.png"
+displayed_sidebar: docs
+description: "Webpack 5 Issues | Documentation - Web3Auth"
+---
+
+---
+
+## Supported Browsers List
+
+Web3Auth SDKs have functions which require BigInt compatability and thus the browsers which are supported are as follows. The browsers are listed
+alongside their versions which are supported.
+
+- Chrome 67+
```

Comment:
Also, what was the browserlist config to get the below result on https://browsersl.ist?